test/python/circuit/library/test_nlocal.py

```diff
@@ -1231,38 +1229,29 @@ def test_fsim_circuit(self):
         parameters = [1] * (num_qubits * (reps + 1) + reps * (1 + num_qubits))
         param_iter = iter(parameters)
 
-        #      ┌───────┐┌─────────┐┌─────────┐        ┌───────┐                   »
-        # q_0: ┤ Rz(1) ├┤0        ├┤0        ├─■──────┤ Rz(1) ├───────────────────»
-        #      ├───────┤│  Rxx(1) ││  Ryy(1) │ │P(1) ┌┴───────┴┐┌─────────┐       »
-        # q_1: ┤ Rz(1) ├┤1        ├┤1        ├─■─────┤0        ├┤0        ├─■─────»
-        #      ├───────┤└─────────┘└─────────┘       │  Rxx(1) ││  Ryy(1) │ │P(1) »
-        # q_2: ┤ Rz(1) ├─────────────────────────────┤1        ├┤1        ├─■─────»
-        #      └───────┘                             └─────────┘└─────────┘       »
-        # «              ┌─────────┐┌─────────┐        ┌───────┐                   »
-        # «q_0: ─────────┤0        ├┤0        ├─■──────┤ Rz(1) ├───────────────────»
-        # «     ┌───────┐│  Rxx(1) ││  Ryy(1) │ │P(1) ┌┴───────┴┐┌─────────┐       »
-        # «q_1: ┤ Rz(1) ├┤1        ├┤1        ├─■─────┤0        ├┤0        ├─■─────»
-        # «     ├───────┤└─────────┘└─────────┘       │  Rxx(1) ││  Ryy(1) │ │P(1) »
-        # «q_2: ┤ Rz(1) ├─────────────────────────────┤1        ├┤1        ├─■─────»
-        # «     └───────┘                             └─────────┘└─────────┘       »
-        # «
-        # «q_0: ─────────
-        # «     ┌───────┐
-        # «q_1: ┤ Rz(1) ├
-        # «     ├───────┤
-        # «q_2: ┤ Rz(1) ├
-        # «     └───────┘
+        #     ┌───────┐┌───────────────┐           ┌───────┐                    »
```

Comment:
the spaces in this figure should also be fixed.

Reply:
Done.